docs/dev-guide/integration/rest-api/content/associate.rst

```diff
@@ -40,7 +40,6 @@ with an importer that supports the type of units being copied.
       }
     },
    'override_config': {
-     'resolve_dependencies: true,
```

Comment:
I also checked in 2.6release branch were this endpoint was still present and as you can see from the paste it did not solve dependencies for package lion http://fpaste.org/236485/52269011/

Anyway i think we are safe to remove this dead and unused code.